test/MimeTest.php

```diff
@@ -152,6 +152,53 @@ public static function dataTestEncodeMailHeaderQuotedPrintable(): array
         // phpcs:enable
     }
 
+    /**
+     * @dataProvider dataTestEncodeMailHeaderQuotedPrintableWithHeaderName
+     */
+    public function testEncodeMailHeaderQuotedPrintableWithHeaderName(
+        string $str,
+        string $charset,
+        string $expectedResult,
+        int $headerLength
+    ): void {
+        $actualResult = Mime\Mime::encodeQuotedPrintableHeader($str, $charset, 78, Mime\Mime::LINEEND, $headerLength);
+        $this->assertEquals($expectedResult, $actualResult);
+    }
+
+    /** @psalm-return array<array-key, array{0: string, 1: string, 2: string, 3: int}> */
+    public static function dataTestEncodeMailHeaderQuotedPrintableWithHeaderName(): array
+    {
+        return [
+            [
+                "xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx",
+                "UTF-8",
+                "=?UTF-8?Q?xxxxx=20xxxxx=20xxxxx=20xxxxx=20xxxxx=20xxxxx=20xxxxx=20?=" . Mime\Mime::LINEEND
+                . " =?UTF-8?Q?xxxxx=20xxxxx=20xxxxx=20xxxxx=20xxxxx=20xxxxx?=",
+                9,
+            ], // Long string with header name size
```

Comment:
Not a problem, but if you use these as array keys, instead of comments, you get valuable PHPUnit output while running (`--testdox` or such) and in case of failures :)

Reply:
Fixed ))